<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandEditor</class>
 <widget class="QFrame" name="CommandEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Text :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leText"/>
   </item>
   <item>
    <widget class="QLabel" name="lCommand">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Command :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="leCommand"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbCommand">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/tools/icons/tools/file.png</normaloff>:/tools/icons/tools/file.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lWorkDir">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Working Directory :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="leWorkDir"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbWorkDir">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/tools/icons/tools/icon.png</normaloff>:/tools/icons/tools/icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lParsers">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parsers :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="pCheckComboBox" name="ccParsers"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cbSkipOnError">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Skip On Error</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pCheckComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">pCheckComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
